packages/streams/udp.js

```diff
@@ -58,7 +58,7 @@ Udp.prototype.pipe = function (pipeTo) {
   if (this.options.outEvent && this.options.port !== undefined) {
     this.options.app.on(this.options.outEvent, function (d) {
       self.debug('sending over udp: %s', d)
-      socket.send(d, 0, d.length, self.options.port, '255.255.255.255')
+      socket.send(d, 0, d.length, self.options.port, self.options.host || '255.255.255.255')
```

Comment:
Should there be a note that you can leave the host empty and it will use 255.255.255.255? Will that ever work, or does the host actually always need to be there?

Reply:
I don't know if that would ever work! I suspect that it won't